src/APL/fileAPL.ts

```diff
@@ -0,0 +1,87 @@
+import debugPkg from "debug";
+import { promises as fsPromises } from "fs";
+
+import { APL, AuthData } from "./apl";
+
+const debug = debugPkg.debug("FileAPL");
+
+export const loadDataFromFile = async (fileName: string): Promise<AuthData | undefined> => {
```

Comment:
For me it looks like a private method that is exported just to be tested? If so, I'd not to - private methods should be tested indirectly through public methods.

(this is problem not using classes, functions tend to leak due to lack of private keywords)